src/components/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from '@chakra-ui/react';
+
+type ButtonProps = {
+  onClick: () => void;
+  buttonText: string;
+};
+
+const ButtonComp = ({ onClick, buttonText }: ButtonProps) => {
```

Comment:
Let's call this `Button` instead of `ButtonComp` ^^

Reply:
When I name the custom component "Button" it has conflicts with the chakra-ui "Button" import. That was why we initially named it "ButtonComp". Would it be okay to name it "button" or something similar so there are no import issues?

Reply:
You can actually rename the Chakra UI import like this: `import { Button as DefaultButton } from '@chakra-ui/react';`! The idea is that you would only import the Chakra UI button when creating your styled button, and everywhere else in the application you'd be importing the new button you created so that you don't need to restyle it every time.